src/utils/openapi.ts

```diff
@@ -121,6 +121,10 @@ export function isPrimitiveType(
   schema: OpenAPISchema,
   type: string | string[] | undefined = schema.type,
 ) {
```

Comment:
I think this file is missing one change, redocExtensions should have a few extra records

Reply:
yes, you are right. Added.